operations/hpe_pdu/hpe_pdu_admin_procedures.md

```diff
@@ -48,22 +73,20 @@ Set up an HPE PDU for administrative use by completing the following tasks:
 * Add the default user
 * Enable Outlet Control
 
-### Procedure
-
 1. Connect to the HPE PDU Web Interface (See [Connect to HPE PDU Web Interface](#connect-to-hpe-pdu-web-interface)) and log in as `admin`.
 
-#### Ensure Redfish is Enabled
+### Ensure Redfish is Enabled
 
 1. Use the **"Settings"** icon (gear in computer monitor in top right corner) to navigate to **"Network Settings"**.
-1. Verify there is a check next to "RESTapi Access", if not, click the **"Edit"** icon (pencil) and enable.
+1. Verify there is a check next to `RESTapi Access`, if not, click the **"Edit"** icon (pencil) and enable.
 
-#### Add Default User
+### Add Default User
 
 1. Use the **"admin"** menu (top right corner) to navigate to **"User Accounts"**.
 1. Click on the **"Add User"** button.
 1. Use the form to add the _username_ and _password_ for the default River user. Assign the role _"Administrator"_ to that user.
```

Comment:
Should we have a step here to query Vault to see what the expected credential of the PDU should be? 

The following is assuming that the hostname of the PDU resolves (which it should, and that means the discovery job has populated vault):
```
ncn-m001:~ # cray scsd bmc creds list --targets x3000m0
[[Targets]]
Xname = "x3000m0"
Username = "root"
Password = "foobar"
StatusCode = 200
StatusMsg = "OK"
```

Reply:
I am not going to add this as it appears that the PDU will need to be discovered before this is populated, and it needs the root user before it can be discovered.